cirrusci.sh

```diff
@@ -15,10 +15,12 @@ if [ ! -z ${HOST_DC+x} ] ; then HOST_DMD=${HOST_DC}; fi
 if [ -z ${HOST_DMD+x} ] ; then echo "Variable 'HOST_DMD' needs to be set."; exit 1; fi
 
 if [ "$OS_NAME" == "linux" ]; then
-  packages="git-core make g++ gdb curl libcurl3 tzdata zip unzip xz-utils"
+  # Needed to bootstrap installation of tzdata for ubuntu 18.04 images
+  ln -snf /usr/share/zoneinfo/UTC /etc/localtime && echo UTC > /etc/timezone
```

Comment:
This isn't needed for LDC CI, which also installs `tzdata`: https://github.com/ldc-developers/ldc/blob/dae61cb8d745642e9eebc85c294c93d2f62c07d9/.cirrus.yml#L58-L72

Reply:
Probably `export DEBIAN_FRONTEND=noninteractive` does it for you.

Reply:
What is/was needed for gdmd only: https://github.com/ldc-developers/ldc/blob/dae61cb8d745642e9eebc85c294c93d2f62c07d9/.cirrus.yml#L144-L145

Reply:
> What is/was needed for gdmd only: https://github.com/ldc-developers/ldc/blob/dae61cb8d745642e9eebc85c294c93d2f62c07d9/.cirrus.yml#L144-L145

You can blame that on Perl. ;-)